integration/ports.go

```diff
@@ -35,6 +35,7 @@ const (
 	PrivateDataBasePort
 	RaftBasePort
 	SBEBasePort
+	ConfigBasePort
```

Comment:
Please keep these sorted in alphabetical order.

Reply:
Done